stdlib/3/smtplib.pyi

```diff
@@ -41,38 +45,42 @@ class SMTP:
     esmtp_features = ...  # type: Any
     source_address = ...  # type: Any
     local_hostname = ...  # type: Any
-    def __init__(self, host=..., port=..., local_hostname=..., timeout=...,
-                 source_address=...): ...
+    def __init__(self, host: str = ..., port: int = ...,
+                 local_hostname: Optional[str] = ..., timeout: Any = ...,
```

Comment:
Timeout should probably be a float; it gets passed to `socket.create_connection`, which is annotated as taking a float.